Rakefile

```diff
@@ -1,6 +1,10 @@
 require "yast/rake"
 
 Yast::Tasks.configuration do |conf|
+  conf.obs_api = "https://api.suse.de/"
+  conf.obs_target = "SLE-12-SP1"
+  conf.obs_sr_project = "SUSE:SLE-12-SP1:GA"
+  conf.obs_project = "Devel:YaST:Head"
```

Comment:
I think we should create a `Devel:YaST:SLE-12-SP1` subproject, just like we have [`Devel:YaST:SLE-12`](https://build.suse.de/project/show/Devel:YaST:SLE-12)...